### ISSUE TITLE
Link to F-Droid return 404

### ISSUE THREAD

**Issue by ghost** [yeriomin/YalpStore#645]

**Comment by sheerun** [comment]
Same issue. Is it taken down?

**Comment by JonnyTech** [comment]
YalpStore has been abandoned for many years, please use [Aurora Store](https://auroraoss.com/)

**Comment by martin-braun** [comment]
@JonnyTech Which doesn't work most of the time, even the nightlies have regular problems. Any better alternative?

**Comment by JonnyTech** [comment]
Nope, lately Google is really cracking down on Aurora, they are busy working on it, keep trying their Nightlies builds.
